referral-form-data-process/lib/sendDataToAPI.ts

```diff
@@ -0,0 +1,192 @@
+      formDataAnswersObject.clientThreeLastName = value.toString();
+    }
+    if (key === "Child 4: Child's First Name") {
+      formDataAnswersObject.clientFourFirstName = value.toString();
+    }
+    if (key === "Child 4: Child's Last Name") {
+      formDataAnswersObject.clientFourLastName = value.toString();
+    }
+    if (key === "Child 5: Child's First Name") {
+      formDataAnswersObject.clientFiveFirstName = value.toString();
+    }
+    if (key === "Child 5: Child's Last Name") {
+      formDataAnswersObject.clientFiveLastName = value.toString();
+    }
+    if (key === "Child 6: Child's First Name") {
+      formDataAnswersObject.clientSixFirstName = value.toString();
+    }
+    if (key === "Child 6: Child's Last Name") {
+      formDataAnswersObject.clientSixLastName = value.toString();
+    }
+    if (key === "Child 7: Child's First Name") {
+      formDataAnswersObject.clientSevenFirstName = value.toString();
+    }
+    if (key === "Child 7: Child's Last Name") {
+      formDataAnswersObject.clientSevenLastName = value.toString();
+    }
+    if (key === "Child 8: Child's First Name") {
+      formDataAnswersObject.clientEightFirstName = value.toString();
+    }
+    if (key === "Child 8: Child's Last Name") {
+      formDataAnswersObject.clientEightLastName = value.toString();
+    }
+  }
+
+  const clientOne = getClientFullName(
+    formDataAnswersObject.clientOneFirstName,
+    formDataAnswersObject.clientOneLastName
+  );
+  const clientTwo = getClientFullName(
+    formDataAnswersObject.clientTwoFirstName,
+    formDataAnswersObject.clientTwoLastName
+  );
+  const clientThree = getClientFullName(
+    formDataAnswersObject.clientThreeFirstName,
+    formDataAnswersObject.clientThreeLastName
+  );
+  const clientFour = getClientFullName(
+    formDataAnswersObject.clientFourFirstName,
+    formDataAnswersObject.clientFourLastName
+  );
+  const clientFive = getClientFullName(
+    formDataAnswersObject.clientFiveFirstName,
+    formDataAnswersObject.clientFiveLastName
+  );
+  const clientSix = getClientFullName(
+    formDataAnswersObject.clientSixFirstName,
+    formDataAnswersObject.clientSixLastName
+  );
+  const clientSeven = getClientFullName(
+    formDataAnswersObject.clientSevenFirstName,
+    formDataAnswersObject.clientSevenLastName
+  );
+  const clientEight = getClientFullName(
+    formDataAnswersObject.clientEightFirstName,
+    formDataAnswersObject.clientEightLastName
+  );
+
+  if (clientOne) {
+    clientsValue.push(clientOne);
+  }
+  if (clientTwo) {
+    clientsValue.push(clientTwo);
+  }
+  if (clientThree) {
+    clientsValue.push(clientThree);
+  }
+  if (clientFour) {
+    clientsValue.push(clientFour);
+  }
+  if (clientFive) {
+    clientsValue.push(clientFive);
+  }
+  if (clientSix) {
+    clientsValue.push(clientSix);
+  }
+  if (clientSeven) {
+    clientsValue.push(clientSeven);
+  }
+  if (clientEight) {
+    clientsValue.push(clientEight);
+  }
```

Comment:
For now I think this is okay but will look to refactor/tidy when the API backend is implemented with postgres. When we are more certain about structure of the data being sent it will make sense to tidy this up.

Reply:
Yeah agree completely, not a fan of this way either but don't have any ideas currently on how to improve it